sdk/core/src/main/java/com/klaviyo/core/config/KlaviyoConfig.kt

```diff
@@ -161,13 +179,25 @@ object KlaviyoConfig : Config {
             KlaviyoConfig.applicationContext = applicationContext as Context
             KlaviyoConfig.debounceInterval = debounceInterval
             KlaviyoConfig.networkTimeout = networkTimeout
-            KlaviyoConfig.networkFlushInterval = networkFlushInterval
+            KlaviyoConfig.networkFlushIntervals = networkFlushIntervals
             KlaviyoConfig.networkFlushDepth = networkFlushDepth
             KlaviyoConfig.networkMaxRetries = networkMaxRetries
 
             return KlaviyoConfig
         }
     }
+
+    enum class NetworkTypes(val position: Int) {
```

Comment:
Not sure if this should live here (where configs store timers based on network types) or in the network monitor (where timers are read based on network types). Guess it depends on if we'd use network types for other operations in the future.

Reply:
Oh ya, I didnt even realize I pluralized that. Gross. Will fix and move to `NetworkMonitor`

Reply:
Also realized I could probably just use `ordinal()` instead of having this passed in `position` value to assign to each enum entry. Can go either way I spose

Reply:
probably the nitpickiest of nits. `ordinal` is cool, hadn't seen that before